### DISCUSSION TITLE
How to make milestone percentage go up?

### DISCUSSION THREAD

**Discussion by kokonut27** [VulcanWM/jasonism#33] [Q&A] [answered]
# Heya!

How do you make your milestone percentage go up higher?

Thanks!

**Comment by VulcanWM** [comment, marked as answer]
You just close issues that you have assigned to that milestones and the milestone percentage goes up.

**Reply by kokonut27** [reply]
Oh, okay! Thanks so much!

**Reply by VulcanWM** [reply]
np!
